tests/test_userstory_39.py

```diff
@@ -5,13 +5,19 @@
 """
 import uuid
 
+import os, tempfile, base64
+from PIL import Image
```

Comment:
can you sort the imports using `isort`? Ask around in the office on how to use/configure isort :)

Reply:
Pretty straightforward "isort tests/test_userstory_39.py" worked well

Reply:
a `setup.cfg` with the configuration would be nice so that everyone uses the same config :-)

Reply:
Alright, basically used the same I found in zrc